conformance/gatewayclass.yaml

```diff
@@ -0,0 +1,7 @@
+kind: GatewayClass
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: gateway-conformance
+spec:
+  controller: istio.io/gateway-controller
```

Comment:
I assume this needs to be configurable?

Reply:
Yep, added some comments to make this configurable. Will eventually likely change this to a generic name and/or remove it altogether, but for now it's helpful when running tests.